docs/sdk/src/reference_docs/frame_runtime_upgrades_and_migrations.rs

```diff
@@ -0,0 +1,135 @@
+//! # Runtime Upgrades
+//!
+//! At their core, blockchain logic consists of
+//!
+//! 1. on-chain state and
+//! 2. a state transition function
+//!
+//! In Substrate-based blockchains, state transition functions are referred to as
+//! [runtimes](https://docs.substrate.io/learn/runtime-development/).
```

Comment:
Link to https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/blockchain_state_machines/index.html instead?

Reply:
IN fact this article seems to be very much ion conjuncture to https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/blockchain_state_machines/index.html and https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/wasm_meta_protocol/index.html.